mixer/pkg/runtime/runtime.go

```diff
@@ -159,14 +159,14 @@ func (c *Runtime) processNewConfig() {
 
 	builder := compiled.NewBuilder(newSnapshot.Attributes)
 	newRoutes := routing.BuildTable(
-		newHandlers, newSnapshot, builder, c.defaultConfigNamespace, log.DebugEnabled())
+		newHandlers, newSnapshot, builder, c.defaultConfigNamespace, true)
```

Comment:
Do we really want this?

Reply:
Sorry. This is highly useful when debugging, but I meant to revert it before checking in.